pkg/wdk/services_chain_block_headers.go

```diff
@@ -46,3 +53,64 @@ type ChainBlockHeader struct {
 	// Represented as a 32-byte hex string with reversed byte order.
 	Hash string
 }
+
+// Hex returns the hexadecimal string representation of the block header.
+// It marshals the block header fields into a byte slice and encodes it as a hex string.
+// Returns an error if marshaling fails.
+func (c *ChainBaseBlockHeader) Hex() (string, error) {
+	bb, err := c.Bytes()
+	if err != nil {
+		return "", fmt.Errorf("failed to marshal chain block header height: %w", err)
+	}
+	return hex.EncodeToString(bb), nil
+}
+
+// Bytes returns the serialized byte representation of the block header.
+// It includes the reversed previous block hash and Merkle root,
+// followed by time, bits, and nonce fields written in little-endian order.
+// Returns an error if any of the fields cannot be parsed or written.
+func (c *ChainBaseBlockHeader) Bytes() ([]byte, error) {
+	hash, err := hex.DecodeString(c.PreviousHash)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert 'previous hash' field into bytes slice: %w", err)
+	}
+
+	root, err := hex.DecodeString(c.MerkleRoot)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert 'merkle root' field into bytes slice: %w", err)
+	}
+
+	var buff bytes.Buffer
```

Comment:
Consider preallocating the buffer - it's trivial in this case, because it has well-known, constant length: `80`.

```suggestion
	buff := bytes.NewBuffer(make([]byte, 0, 80)) 
```